WiinUPro/Windows/DevicePrefsWindow.xaml

```diff
@@ -32,5 +32,18 @@
         <Label x:Name="labelExtProfile" Content="Extension Specific Profile:" HorizontalAlignment="Left" Margin="10,114,0,0" VerticalAlignment="Top" FontWeight="Bold" Foreground="{StaticResource TextHeader}"/>
         <TextBox x:Name="extProfile" Height="23" Margin="10,167,10,0" TextWrapping="Wrap" Text="" VerticalAlignment="Top" Foreground="#FF444444"/>
         <Button x:Name="btnExtProfile" Content=". . ." HorizontalAlignment="Right" Margin="0,144,10,0" VerticalAlignment="Top" Width="32" Style="{StaticResource BasicButton}" Click="btnExtProfile_Click"/>
+        <TextBlock Text="№ Specific Profile" HorizontalAlignment="Left" Margin="119,68,0,0" VerticalAlignment="Top" Foreground="{DynamicResource BackgroundMain}" ToolTip="Profiles whose names end with &quot;_№&quot; (for example &quot;Nunchuk_Xinput_IR_1.wup&quot;) will be automatically switched between depending on the currently connected devices.&#x0a;The number should correspond with the selected rumble controller of the profile. This functionality is only available for Nintendo controllers.">

```

Comment:
Can we split this line up, it's pretty long. I haven't been super good about that but I'm gonna eventually try and tackle optimization opportunities as I run into them.